task/buildah-pr/0.1/buildah-pr.yaml

```diff
@@ -56,14 +61,17 @@ spec:
         value: vfs
     - name: push
       image: $(params.BUILDER_IMAGE)
-      workingDir: /workspace/source
-      command: ['buildah', 'push', '--tls-verify=$(params.TLSVERIFY)', '$(resources.outputs.image.url)', 'docker://$(resources.outputs.image.url)']
+      workingDir: $(workspaces.source.path)
```

Comment:
Same here, this shouldn't be changed in the `-pr` task. (or maybe using `$(resources.inputs.source.path)` if it works)

Reply:
Done, replaced with `$(resources.inputs.source.path)`